### ISSUE TITLE
[Bug] Ship phases through custom landing pad

### ISSUE THREAD

**Issue by Sullos** [tsunamayo/Starship-EVO#2336]
### Short Description of the issue:
I built a custom landing pad to test for my station building. I set the shipyard bounds, and got into my ship, targeted the landing pad, and requested landing. Once I got into the landing zone, the ship was magnetized and started lowering down onto the pad, the counter in the corner of the screen showing how close to the pad I am got to 0m then started going negative as my entire ship phased through the landing pad and came out the bottom.

Basically, I tried landing on a custom pad, and my ship just phased through it and kept going down because it still didn't think it had made contact with the pad yet.

### Environment
- Starship EVO version: 20w23c
- Graphics Card: 2080 Super OC

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4728425/output_log.txt)

**Comment by Xenoprophet-Nail** [comment]
I've had a similar issue with landing on the Torus, as in if I land anywhere my ship sinks into the terrain, I wonder if the issues are related? 
Environment:
Starship EVO Version: 20w23c
Graphics Card Nvidia Geforce Gtc 1060 6gb ram

Pre Station update (Mind the damage please)
![20200530063541_1](https://user-images.githubusercontent.com/51257864/83735728-bdb1d780-a61e-11ea-84d3-b41d1888d49a.jpg)

Post station update: 
![20200531100610_1](https://user-images.githubusercontent.com/51257864/83735815-d5895b80-a61e-11ea-9b95-048e8740c640.jpg)
![20200603064231_1](https://user-images.githubusercontent.com/51257864/83735824-d8844c00-a61e-11ea-8dde-9503d331a542.jpg)

**Comment by tsunamayo** [comment]
@Sullos is fixed for 20w23d. But @Xenoprophet-Nail thats seems to be a different one! Do you get that with all the ship?

**Comment by Xenoprophet-Nail** [comment]
it seems to only be one ship I have so far, then again it's the one that I land on the torus that has a gravity block on it, my small fighter sat just fine on the surface.. I'll include my ship here in case something weird is going on.


[Vortac xHBt.zip](https://github.com/tsunamayo/Starship-EVO/files/4735536/Vortac.xHBt.zip)

**Comment by tsunamayo** [comment]
@Xenoprophet-Nail issue fixed for 20w23d. Glad you mention that one, it was something different but very ugly! Cheers
